### PR TITLE
feat(atomic): remove atomic-insight-breadbox from docs, warn when implementing...

### PR DESCRIPTION
**Pull request by liviarett**
breadbox with non-search

I actually didn't find any mention of the breadbox in the Figma file, so removed it at least for now!

https://coveord.atlassian.net/browse/KIT-1907

### CODE REVIEW

**Review by ThibodeauJF — CHANGES_REQUESTED** [review]
This will break implementations using `atomic-external`

File: packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx [hunk 1]
```diff
@@ -84,12 +85,18 @@ export class AtomicBreadbox implements InitializableComponent {
   private breadcrumbShowLessFocus!: FocusTargetController;
 
   public initialize() {
-    this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
-    this.facetManager = buildFacetManager(this.bindings.engine);
-
-    if (window.ResizeObserver) {
-      this.resizeObserver = new ResizeObserver(() => this.adaptBreadcrumbs());
-      this.resizeObserver.observe(this.host.parentElement!);
+    if (!closest(this.host, 'atomic-search-interface')) {
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx:88]: Maybe the bindings could contain what type of interface you're connected to (search/insight)

**Review by olamothe — COMMENTED** [review]
I understand the JS doc removal, but why explicitly add code for this specific component that verifies that the parent is a search interface, and not for all the other search components ?

The "convention" we went with is that implementers should be using components that match for `insight` (ie: will contain the `insight` prefix in the component tag name).

The changes in atomic-breadbox seems superfluous ?

**Comment by liviarett** [comment]
@olamothe you're right, I think that because it kind of works with the wrong interface already without changing anything, I wanted to make sure it was clear that it wasn't meant to be used like that. but I agree that it should be handled somewhere else and in a more generic way!

**Review by btaillon — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]

**Pull request merged**